### DISCUSSION TITLE
matRad optimizer finished only 1 iterates

### DISCUSSION THREAD

**Discussion by MP1234567899** [e0404/matRad#746] [Q&A]
Hi,
I used 4DCT data with 10 phases for robust optimization in matRad. However, I encountered an issue when planning in the average phase. The optimization only ran for one iteration and produced warnings and errors in the MATLAB command window.  I suspect there might be an error with how the image was exported from my TPS, since optimization works for other individual phases from the same 4DCT data.
Error showed about using caxis; limits must be a 2-element , matRad_plotColorbar, matRad_ViewingWidget/UpdatePlot 
I get an error when using gantry angles of xx and xx degrees without including couch rotation. However, if I add couch rotation while keeping the same gantry angles, the optimization completes, but the resulting dose distribution is unusual.

Thank you.

**Comment by wahln** [comment]
This is currently discussed in #747
